src/fidesops/service/connectors/saas_query_config.py

```diff
@@ -123,52 +124,6 @@ def generate_requests(
 
         return request_params
 
-    def map_param_values(
```

Comment:
Moved `map_param_values` to `saas_util` to be able to use it freely without having to refer to or instantiate an unecessary SaaSQueryConfig.